project/profiles/capacity4more/modules/c4m/user/c4m_user/actions/c4m_user_contact_multiple.action.inc

```diff
@@ -0,0 +1,173 @@
+<?php
+
+/**
+ * @file
+ * Callback & forms related to the user contact multiple.
+ */
+
+/**
+ * Implements hook_action_info().
+ *
+ * A custom action to contact a multiple users with a mail action.
+ */
+function c4m_user_contact_multiple_action_info() {
+  $group = c4m_og_current_group();
+  // If not within OG context, don't declare action.
+  if (!$group) {
+    return array();
+  }
+  // If not Site Admin, Group Admin or Group Owner, don't declare action.
+  if (!_c4m_features_og_members_is_power_user($group)) {
+    return array();
+  }
+  // Site Admin, Group Admin or Group Owner => declare action.
```

Comment:
1. Move this comment to the docs of the function.
2. Leave a line spacing above the `return` command.